presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java

```diff
@@ -95,5 +99,22 @@ public void testExplainOfCreateTableAs()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan("EXPLAIN ", query, LOGICAL));
     }
 
+    @Test
```

Comment:
LG. Just one question, why not putting it to AbstractTestDistributedQueries.java like other CTE tests?

Reply:
Hive connector is the only connector which supports creation of temporary tables (which is done in physicalCteOptimizer) the .`setSystemProperty(CTE_PARTITIONING_PROVIDER_CATALOG, "hive")` flag for eg